package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@epickris/zpl",
-    "version": "0.2.1",
+    "version": "0.3.1",
```

Comment:
Do not change the package version, this will be changed when a release is made.

Reply:
I just update this